File: tests/syn/kotlin/01-kitchen.kt
package demo
/// docs
// ordinary line comment
/* outer /* nested TODO */ done */
@JvmInline value class UserId(val raw: Long)
sealed interface Result
data class User(val name: String, val age: Int)
fun greet(user: User?): String {
    val n = 0xCA_FEuL + 1.5e2F
    return if (user?.name != null) "hello $user ${user.name ?: "none"}" else "missing\\n"
}
val char = '\n'
val truth = true
val absent = null
*/

File: tests/syn/swift/01-kitchen.swift
import Foundation
/// docs
// ordinary line comment
/* outer /* nested TODO */ done */
@available(macOS 14, *)
public final class Box<T> {
    let value: T
    weak var owner: Any?
    init(_ value: T) { self.value = value }
    func render(_ n: Int) -> String {
        let hex = 0xCA_FE + 1.5e2
        if n >= 0 && true { return "value \(value) nested \((n + 1))\n" }
        return String(describing: nil)
    }
}
*/
